packages/ckeditor5-engine/tests/view/downcastwriter/writer.js

```diff
@@ -122,6 +122,17 @@ describe( 'DowncastWriter', () => {
 			expect( element.id ).to.equal( 'bar' );
 			assertElementAttributes( element, attributes );
 		} );
+
+		it( 'should not treat priority 0 as no priority set', () => {
+			const element = writer.createAttributeElement( 'foo', attributes, { priority: 0, id: 'bar' } );
+
+			expect( element.is( 'attributeElement' ) ).to.be.true;
```

Comment:
You can skip checking anything else than `element.priority` here. Other properties are handled by relevant assertion, your point here is to check that priority has been set to `0`.